CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## master (unreleased)
 
+### New features
+* [#446](https://github.com/rubocop/rubocop-rails/issues/446): Add new `Rails/RequireDependency` cop. ([@tubaxenor][])
```

Comment:
Can you add the blank line?

```suggestion
### New features

* [#446](https://github.com/rubocop/rubocop-rails/issues/446): Add new `Rails/RequireDependency` cop. ([@tubaxenor][])
```

Reply:
Did someone say... `rubocop/rubocop-changelog`? 😄

Reply:
It's not all automated and follows convention.

Reply:
And I'm planning to introduce the new changelog entry format introduced in RuboCop core in near future (maybe ASAP), but it is not ready yet.
https://github.com/rubocop/rubocop/blob/v1.11.0/CONTRIBUTING.md#changelog-entry-format